### PR TITLE
1 new Language added (Hindi-"hi")

### PR DESCRIPTION
**Pull request by markandey1414**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description
A new language "Hindi-hi" was added to the path `lib\l10n\arb\app_hi.arb`. It was also updated into the supportedLocales.

## Featured Covered in this PR

- [x] Task 1
- [ ] Task 2

## Related Tickets & Documents

- Related Issue #
- Closes #

## Screenshots, Recordings

*Please replace this line with instructions on how to test your changes, or any screenshots or recording you can attach here.*

## Tested Feature??

- [x] In Real Device.
- [ ] In Emulator

### CODE REVIEW

File: lib/app/view/app.dart [hunk 1]
```diff
@@ -107,7 +107,7 @@ class _AppViewState extends State<AppView> {
           navigatorKey: _navigatorKey,
           debugShowCheckedModeBanner: false,
           title: "My Dairy",
-          supportedLocales: const [Locale('en')],
+          supportedLocales: const [Locale('en')], const [Locale('hi')],
```
Comment (SankethBK) [inline lib/app/view/app.dart:110]: there is a syntax error in this line, change it to 
```
          supportedLocales: const [Locale('en'), Locale('hi')],
```

File: lib/l10n/arb/app_hi.arb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  },
+
+
+  "pageNotFound":"पृष्ठ नहीं मिला",
+  "settings":"समायोजन",
+  "enterNewEmail":"नया ईमेल दर्ज करें",
+  "enterRegisteredEmail":"पंजीकृत ईमेल दर्ज करें",
+  "enterCurrentPassword":"वर्तमान पासवर्ड दर्ज करें",
+  "resetPassword":"पासवर्ड रीसेट",
+  "autoSync":"Auto sync",
```
[the remark this answers is not in the document]
Comment (SankethBK) [reply lib/l10n/arb/app_hi.arb:14]: If it becomes less meaningful on translating, let it be

**Comment by SankethBK** [comment]
Minor comments, rest all looks good

File: lib/l10n/arb/app_hi.arb (hunk 2 above)
[the remark this answers is not in the document]
Comment (markandey1414) [reply lib/l10n/arb/app_hi.arb:14]: yes doing it.

File: lib/app/view/app.dart (hunk 1 above)
Comment (markandey1414) [reply lib/app/view/app.dart]: done!

**Review by SankethBK — APPROVED** [review]
LGTM, thanks for contributing

**Pull request merged**